app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.scss

```diff
@@ -75,3 +75,20 @@
     display: none;
   }
 }
+
+.popover {
+  font-family: $FONT-ROBOTO-REGULAR;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
These properties (`font-style`, `font-weight`) are normal by default for all fonts, please remove them.